### ISSUE TITLE
[QUESTION] Multiple FCM keys

### ISSUE THREAD

**Issue by victorpre** [codedge-llc/pigeon#153]
Hello,
Is it possible to configure multiple fcm keys?

In the README documentation there is:

```
config :pigeon, :fcm,
  fcm_default: %{
    key: "your_fcm_key_here"
  }
```

I would like to know if there is a way to configure multiple fcm keys or if I can change this keys during Runtime

**Comment by hpopp** [comment]
Addressed in #152. New documentation is out on hex.

**Comment by victorpre** [comment]
Thx @hpopp
